Detectors/TRD/calibration/src/CalibratorVdExB.cxx

```diff
@@ -126,6 +126,18 @@ void CalibratorVdExB::initProcessing()
   opt.SetTolerance(.001);
   mFitter.Config().SetMinimizerOptions(opt);
 
+  // Set tree branches
+  if (mOutTree) {
+    mOutTree->Branch("lorentzAngle", &mFitFunctor.laPreCorr);
+    mOutTree->Branch("vDrift", &mFitFunctor.vdPreCorr);
+    //mOutTree->Branch("residuals", &mFitFunctor.profiles);
```

Comment:
ROOT does not support writing this _yet_.
Any idea how to get this in the tree without making a completely new object?

Reply:
Hm, `mFitFunctor.profiles` is an array of unique pointers to TProfile objects. I don't think that ROOT supports this. Maybe it would work with a loop, adding one branch per TProfile. Something like `mOutTree->Branch(fmt::format("residuals_{}", iDet), &(mFitFunctor.profiles[iDet]));`